lib/src/cli/arguments_builder/models/parsed_arguments.dart

```diff
@@ -0,0 +1,21 @@
+import '../../../analyzers/lint_analyzer/metrics/models/metric_value_level.dart';
+
+class ParsedArguments {
+  final String rootFolder;
+  final String reporterName;
+  final String reportFolder;
+  final MetricValueLevel? metricValueLevel;
```

Comment:
@incendial `metricValueLevel` looks not clear, try to rename to `maximumAllowedLevel`

Reply:
Done.